setup.py

```diff
@@ -112,8 +112,14 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'matplotlib', 'scipy', 'poppy', 'astropy'],
-      setup_requires=['numpy', 'astropy'],
+      install_requires=[
+          'numpy>=1.10.0',
+          'matplotlib>=1.5.3',
```

Comment:
Agreed we should only test on the latest matplotlib. Couldn't we set it up to test on the latest but have the install requires set to allow anything 1.5.0 or newer?  Maybe anaconda & astroconda is increasingly smoothing the installation process enough that I'm over thinking things - but from past experience I predict headaches from some astronomers if trying to install webbpsf requires updating a whole bunch of the core scipy stack.   I dunno. 

Perhaps @jhunkeler can advise on best practices here?